src/Kentico.Xperience.Algolia/Indexing/IAlgoliaIndexService.cs

```diff
@@ -4,32 +4,19 @@
 
 using Algolia.Search.Clients;
 
-using Kentico.Xperience.Algolia.Models;
-
-namespace Kentico.Xperience.Algolia.Services
+namespace Kentico.Xperience.Algolia.Indexing
```

Comment:
```suggestion
namespace Kentico.Xperience.Algolia.Indexing;
```

Use file scoped namespace

Reply:
Changed